test/unit/ngsiService/active-devices-test.js

```diff
@@ -352,6 +352,46 @@ describe('Active attributes test', function() {
         });
     });
 
+    describe('When the IoTA gets a set of values with a TimeInstant which are not in ISO8601 format', function() {
+        var modifiedValues;
+
+        beforeEach(function(done) {
+            var time = new Date(1438760101468); // 2015-08-05T07:35:01.468+00:00
+
+            modifiedValues = [
+                {
+                    name: 'state',
+                    type: 'Boolean',
+                    value: 'true'
+                },
+                {
+                    name: 'TimeInstant',
+                    type: 'ISO8601',
+                    value: '2015-12-14T08:06:01 00.46Z'
```

Comment:
What about using exaclty the one that raised the error at #679? I mean:

```
2018-10-05T11:03:56 00:00Z
```

Reply:
Fixed with 171064daac4c14a3e985453dd040294410ffba2c